src/control/cmd/daos_agent/infocache_test.go

```diff
@@ -48,7 +48,10 @@ func TestInfoCacheInitNoScanResults(t *testing.T) {
 	enabled := atm.NewBool(true)
 	scanResults := []netdetect.FabricScan{}
 	aiCache := attachInfoCache{log: log, enabled: enabled}
-	err := aiCache.initResponseCache(&mgmtpb.GetAttachInfoResp{}, scanResults)
+	netCtx, err := netdetect.Init()
+	defer netdetect.CleanUp(netCtx)
+	common.AssertTrue(t, err == nil, "failed to init netdetect context")
```

Comment:
I think it'd be worthwhile to print the error in these cases, something like:
```suggestion
	if err != nil {
		t.Fatalf("failed to init netdetect context: %s", err)
	}
```

Reply:
Done.